framework/src/splits/Split.C

```diff
@@ -49,12 +49,6 @@ Split::validParams()
   params.addParam<MooseEnum>(
       "schur_pre", SchurPreEnum, "Type of Schur complement preconditioner matrix");
 
-  MooseEnum SchurAInvEnum("diag lump", "diag");
-  params.addParam<MooseEnum>(
-      "schur_ainv",
-      SchurAInvEnum,
-      "Type of approximation to inv(A) used when forming S = D - C inv(A) B");
```

Comment:
what motivates this?

Reply:
The option is never used by PETSc. It's a red herring having the option available to users